test/bugs-reported/bug3.chpl

```diff
@@ -0,0 +1,8 @@
+proc max(a:int, b:int): int{
```

Comment:
Yeah, I noticed the single statement if..else. But, I felt, in some scenarios as explained in the JIRA issue, the conditional operators may be more beneficial. Pardon me, if you still feel that if..else can handle all these.